### DISCUSSION TITLE
What are the differences btwn the standard and the pro version?

### DISCUSSION THREAD

**Discussion by bluelephant825** [rsayfutdinov/Pikka#20] [Q&A] [answered]
Hi Ruslan,

I couldn't find the difference between the free and the paid (via in-app purchases) version in the description in the Mac App Store: [link](https://apps.apple.com/us/app/pikka-color-picker/id1195076754?mt=12)

I suggest that you write a short note so that users know what they are paying for.

Your app looks very nice. I'll try to provide some feedback when I have some more time.

Good luck with your app and cheers.
Olivier

**Comment by rsayfutdinov** [comment, marked as answer]
Hi Oliver,

I'm glad you liked the app. 

I'm trying to keep the core functionality like picking colors from the screen or creating colors free for all users.

Here is the list of PRO features:
- iCloud sync
- Creating more than 3 folders in the library
- saving generated color palette
- saving color shades, mono, analogous, complementary, and triad schemes
- exporting palettes to external files
- edit color values by entering hex, rgb, hsl, hsb values in text fields.

I will add it to the AppStore description. 

I'm actively working on Pikka and more PRO features are coming soon. 
And I'm open to any kind of feedback. Hope to hear from you soon :)

Regards, 
Ruslan
